### PR TITLE
Fix `unit.test_doc test` for Windows

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
This test fails on Windows because there is no Grep in Windows and the file paths were formatted for Unix.

Use findstr instead of grep on Windows
Use `os.linesep` for file paths

### What issues does this PR fix or reference?
https://github.com/saltstack/salt-jenkins/issues/439

### Tests written?
Test

### CODE REVIEW

File: tests/unit/test_doc.py [hunk 1]
```diff
@@ -43,25 +52,32 @@ def test_check_for_doc_inline_markup(self):
...
 
             # Set up test return dict
             if test_ret.get(key) is None:
-                test_ret[key] = [val.lstrip()]
+                test_ret[key] = [val.strip()]
             else:
-                test_ret[key].append(val.lstrip())
+                test_ret[key].append(val.strip())
+
+        print('*' * 68)
```
Comment (rallytime) [inline tests/unit/test_doc.py]: Please remove this debugging print.

**Comment by rallytime** [comment]
@twangboy There are some lint failures: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/13740/violations/file/tests/unit/test_doc.py/

File: tests/unit/test_doc.py (hunk 1 above)
Comment (twangboy) [inline tests/unit/test_doc.py]: Doh!

**Comment by twangboy** [comment]
Fixed lint

**Pull request merged**